releasenotes/notes/channel-module-deprecations-in-pulse-ffc384b325b077f1.yaml

```diff
@@ -0,0 +1,23 @@
+---
+other:
```

Comment:
This should be an `upgrade` note not `other`. It's something users will need to know when they upgrade from 0.12.x to 0.13.0

Reply:
ah is that what upgrade means? not "this is bigger and better than before!" 😂 thanks matt!